api/osrd_infra/models/train_schedule.py

```diff
@@ -27,9 +25,6 @@ class TrainScheduleModel(models.Model):
     )
     speed_limit_composition = models.CharField(max_length=128, null=True)
     options = models.JSONField(null=True, blank=True, validators=[PydanticValidator(TrainScheduleOptions)])
-
-    # Simulation outputs
-    mrsp = models.JSONField(validators=[PydanticValidator(MRSP)])
-    base_simulation = models.JSONField()
-    eco_simulation = models.JSONField(null=True)
-    modes_and_profiles = models.JSONField(default=list)
+    simulation_output = models.OneToOneField(
+        SimulationOutput, on_delete=models.CASCADE, related_name="train_schedule", null=True
+    )
```

Comment:
The new class has all the inputs directly at the root level, and the outputs in a new field at the same level?

This causes some weird semantics in other place (see other comments), and it isn't the best at splitting the inputs/outputs: the output is now "part" of the inputs.

I think I would have preferred if we had "inputs" and "outputs" classes, with this TrainSchedule combining one instance of each.

Reply:
not sure we need to have input and output classes, I think `TrainSchedule` and `SimulationOutput` are enough for now. I agree with you on the point that the relation was in an incorrect order before sorry for that I have changed it.